### PR TITLE
Adding us-in-boone scraper based on JailTracker

### PR DESCRIPTION
**Pull request by ajsecord**
Tested with `python -m recidiviz.tools.run_scraper --region us_in_boone --num_tasks 30 --lifo`.

### CODE REVIEW

File: queue.yaml [hunk 1]
```diff
@@ -1055,3 +1055,15 @@ queue:
...
+# us-in-boone-scraper - Boone County, Indiana
+- name: us-in-boone-scraper
+  mode: push
+  rate: 5/m
+  bucket_size: 2
+  max_concurrent_requests: 3
+  retry_parameters:
+    min_backoff_seconds: 5
+    max_backoff_seconds: 300
+    task_age_limit: 3d
```
Comment (colincadams) [inline queue.yaml]: be sure to drop this line before you merge, we realized that it was messing up our retry logic so we removed it from all of the others a couple days ago.

File: recidiviz/ingest/scrape/regions/us_in_boone/us_in_boone_scraper.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# =============================================================================
+
+"""Scraper implementation for us_in_boone."""
+from recidiviz.ingest.scrape.vendors.jailtracker import JailTrackerScraper
+
+
+class UsInBooneScraper(JailTrackerScraper):
+    """Scraper implementation for us_in_boone."""
+    def __init__(self, mapping_filepath=None):
```
Comment (colincadams) [inline recidiviz/ingest/scrape/regions/us_in_boone/us_in_boone_scraper.py]: you should be able to just remove the `mapping_filepath` parameter entirely.

File: queue.yaml (hunk 1 above)
Comment (ajsecord) [reply queue.yaml]: Done.

File: recidiviz/ingest/scrape/regions/us_in_boone/us_in_boone_scraper.py (hunk 2 above)
Comment (ajsecord) [reply recidiviz/ingest/scrape/regions/us_in_boone/us_in_boone_scraper.py]: Done.

**Comment by ajsecord** [comment]
Do not merge, blocked on new enum values.

**Comment by ajsecord** [comment]
Tested with `python -m recidiviz.tools.run_scraper --region us_in_boone --num_tasks 30 --lifo`.

Blocked on #939, but mergeable after that.

**Comment by ajsecord** [comment]
Ready to merge if looking good. Updated to HEAD on master and tested again.

**Pull request merged**